code/modules/mob/living/basic/lavaland/goliath/tentacle.dm

```diff
@@ -41,16 +47,23 @@
 /// Grab everyone we share space with. If it's nobody, go home.
 /obj/effect/goliath_tentacle/proc/grab()
 	for (var/mob/living/victim in loc)
-		if (victim.stat == DEAD || HAS_TRAIT(victim, TRAIT_TENTACLE_IMMUNE))
+		if (victim.stat == DEAD || HAS_TRAIT(victim, TRAIT_TENTACLE_IMMUNE) || caster?.faction_check_mob(victim))
```

Comment:
Was there a faction check previous to my rework? I kind of _like_ goliaths being able to friendly fire other mobs.

Reply:
it was here before but we can just have goliaths not pass caster maybe